src/cogs/gamble.py

```diff
@@ -0,0 +1,152 @@
+import random
+import time
+
+from discord.ext import commands
+import discord
+from discord.ext.commands import Context
+from discord import Member
+from datetime import datetime
+
+from src.util.user_database import *
+
+
+daily_amount = 10
+
+class Gamble(commands.Cog, name="Gamble"):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.Cog.listener()
+    async def on_command_error(self, ctx: Context, error):
+        try:
+            if isinstance(error, commands.MissingRequiredArgument):
+                await ctx.send('Invalid input.')
+            if isinstance(error, commands.MissingPermissions):
+                await ctx.send('You do not have permission to use this command.')
+            if isinstance(error, commands.BadUnionArgument):
+                await ctx.send('Invalid input.')
+        except Exception as error:
+            print(error)
+
+    @commands.hybrid_command(with_app_command=True)
+    async def bet(self, ctx: Context, amount_to_bet: int):
+        """
+        Place a bet (50/50). Enter the amount to bet.
+        """
+        user_database_check_if_user_exists_otherwise_add(ctx.author.id)
+
+        if amount_to_bet == None:
+            await ctx.send('You need to enter the amount to bet.', ephemeral=True)
+            return
+    
+        amount_in_wallet = user_database_get('money', ctx.author.id)
+
+        if amount_to_bet <= 0:
+            await ctx.send('You must enter a positive number greater than 0.', ephemeral=True)
+            return
+        if amount_to_bet > amount_in_wallet:
+            await ctx.send('You don\'t have enough money.', ephemeral=True)
+            return
+
+        won = random.randint(1, 2) == 1
+
+        # Update wallet.
+        amount_result = amount_in_wallet + (amount_to_bet if won else -amount_to_bet)
```

Comment:
I really dislike `(amount_to_bet if won else -amount_to_bet)`. I think it's better to be explicit here, even though it's a little more verbose